Website/src/rendering/src/assets/css/layout/sections.css

```diff
@@ -76,12 +76,38 @@
     @apply bg-white;
   }
 
+  &--dark-pattern,
+  &--light-pattern {
+    @apply relative;
+
+    &::before,
```

Comment:
There is a small gap on the left from the xl breakpoint and up.

![image](https://user-images.githubusercontent.com/96472119/167395076-435832b9-0ce7-4bf3-9033-8c2c5c296d3d.png)

Reply:
This is still there, only for the speakers section and from xl and up.

Reply:
This is still there, only for the speakers section and from xl and up.